src/pages/components/CertificateManager.js

```diff
@@ -33,7 +33,7 @@ const CertificateViewPostSwitcher = () => {
       } catch (e) {
         alert(e)
       }
-      const embodiedCo2 = res?.embodied_co2 != null && res?.embodied_co2 > 0
+      const embodiedCo2 = res?.embodied_co2 != null && res?.embodied_co2 >= 0
```

Comment:
What happens when co2 is negative?

Reply:
It is hard enough to perfectly offset Carbon ( for anything , from a factory to a power-plant ) .. Zero Carbon is already hard to achieve but I have added that option in anyway .. but Negative Carbon ... I don't see that happening any time soon ..

Reply:
Carbon Capture Utilisation and Storage ;) (CCUS)